src/main/kotlin/org/stellar/walletsdk/util/GsonUtils.kt

```diff
@@ -18,51 +12,5 @@ object GsonUtils {
 
   private fun builder(): GsonBuilder {
     return GsonBuilder()
-      .registerTypeAdapter(Instant::class.java, InstantConverter())
-      .registerTypeAdapter(Duration::class.java, DurationConverter())
-  }
-}
-
-class DurationConverter : JsonSerializer<Duration?>, JsonDeserializer<Duration?> {
```

Comment:
These methods were crashing the Android app. Removing them as we're not currently using them.